coding/3sum/meta.yml

```
@@ -0,0 +1,23 @@
+---
+identifier: 3Sum
+category: coding
+subcategory: "array"
+difficulty: medium
+description:
+  en: README_en.md
+solution:
+  en: SOLUTION_en.md
+starter_code:
+  c: "starter.c"
```

if only js and python files are provided, you can remove other languages here:
```suggestion
starter_code:
  python: "starter.py"
  javascript: "starter.js"
```

---

cool. Just so I understand, in the meta.yml, i should only include starter_code for languages that I provide? In future contributions, are you looking for starter_code for all languages?  Or just will you fill in the rest?  Thanks!

---

You can fill in as many languages as you can, and the community can always add in more in the future.